src/main/java/io/orangebeard/listener/OrangebeardTestSystemListener.java

```diff
@@ -152,240 +170,136 @@ public void close() {
-                metaData.put("tags", suitePageData.getAttribute(WikiPageProperty.SUITES));
+                metaData.add(new Attribute("tags", suitePageData.getAttribute(WikiPageProperty.SUITES)));
             }
         }
         return metaData;
     }
 
-    private Maybe<String> startSuite(Maybe<String> parentId, String shortName, Map<String, String> metaData) {
-        StartTestItemRQ rq = getSuite(shortName, metaData);
-        Maybe<String> suiteId = launch.startTestItem(parentId, rq);
-        logger.info("[Orangebeard] suite {} started in {} with id {}", shortName, parentId, suiteId.blockingGet());
-        return suiteId;
+    private UUID startSuite(UUID parentId, String shortName, Set<Attribute> attributes) {
+        StartTestItem suite = new StartTestItem(runContext.getTestRun(), shortName, TestItemType.SUITE, null, attributes);
+        return orangebeard.startSuite(parentId, suite);
     }
 
     private void stopAllSuites() {
-        for (Maybe<String> suiteId : context.getAllSuiteIds()) {
+        for (UUID suiteId : runContext.getAllSuiteIds()) {
             stopSuite(suiteId);
         }
     }
 
-    private void stopSuite(Maybe<String> suiteId) {
-        final FinishTestItemRQ rq = new FinishTestItemRQ();
-        rq.setEndTime(Calendar.getInstance().getTime());
-        launch.finishTestItem(suiteId, rq);
-    }
-
-    private StartLaunchRQ getStartLaunchRQ(TestPage testPage) {
-        StartLaunchRQ startLaunchRQ = new StartLaunchRQ();
-        startLaunchRQ.setMode(Mode.DEFAULT);
-        startLaunchRQ.setStartTime(Date.from(Instant.now()));
-        startLaunchRQ.setName(getLaunchNameFromProperties(testPage));
-        startLaunchRQ.setAttributes(getLauchAttributes());
-        return startLaunchRQ;
+    private void stopSuite(UUID suiteId) {
+        FinishTestItem item = new FinishTestItem(runContext.getTestRun(), null, null, null);
+        orangebeard.finishTestItem(suiteId, item);
     }
 
-    private void reportStartLaunchToAugmented(String launchUUID) {
-        String endpoint = getEndpointFromProperties();
-        String projectId = System.getenv("projectId");
-        String jobId = "jenkins-" + System.getenv("JOB_NAME") + "-" + System.getenv("BUILD_ID");
-        String token = getToken();
 
-        if (endpoint == null) {
-            logger.warn("[Orangebeard] Cannot find " + PROP_AUGMENTED_ENDPOINT + " in " + PROPERTY_FILE_NAME + ". Nothing will be reported to the augmented endpoint!");
-        }
-        if (token == null) {
-            logger.warn("[Orangebeard] Cannot find " + AUGMENTED_TESTING_TOKEN + " in environment variables or in properties. Nothing will be reported to the augmented endpoint!");
-        }
-        if (projectId == null || launchUUID == null) {
-            logger.warn("[Orangebeard] Project id {}, Launch id {} and/or job id {} is null", projectId, launchUUID, jobId);
-            return;
+    @SneakyThrows
```

Comment:
wat is deze?

Reply:
mooie lombok annotatie... anders moet ik een exceptie gaan afhandelen voor het geval localhost een unknown host is. Ik achtte die kans niet heel groot